opencti-platform/opencti-front/src/utils/Localization.js

```diff
@@ -2384,8 +2384,8 @@ const i18n = {
       'Update an artifact': 'Modifier un artefact',
       'Create an infrastructure': 'Créer une infrastructure',
       'Update an infrastructure': 'Modifier une infrastructure',
-      'Create a threat actor': 'Créer un acteur',
-      'Update a threat actor': 'Modifier un acteur',
+      'Create a threat actor group': 'Créer un groupe d\'acteur',
+      'Update a threat actor group': 'Modifier un groupe d\'acteur',
```

Comment:
Missing `s` at the end of `Créer un groupe d\'acteur`  (Créer un groupe d\'acteur**s**)